frontend-admin/src/app/views/organization/organization.controller.coffee

```diff
@@ -24,8 +25,54 @@
   # Get the organization
   MnoeOrganizations.get($stateParams.orgId).then(
     (response) ->
-      vm.organization = response.data
+      vm.organization = response.data.plain()
       vm.organization.invoices = $filter('orderBy')(vm.organization.invoices, '-started_at')
+      vm.updateStatus()
   )
 
+  vm.updateStatus = ->
+    vm.status = {}
+    _.map(vm.organization.active_apps,
+      (app) ->
+        vm.status[app.nid] = MnoAppsInstances.isConnected(app)
+    )
+    vm.hasDisconnectedApps = false of _.countBy(vm.status)
```

Comment:
`_.countBy(vm.status)` return an associative array with the key being the status and the value the count.

https://lodash.com/docs/4.16.6#countBy